packages/flutter/lib/src/services/text_input.dart

```diff
@@ -431,6 +431,15 @@ enum TextCapitalization {
   none,
 }
 
+enum ObscureTextBehavior {
+  /// show all text
```

Comment:
We need full sentences for these, so maybe change to:

Indicates all text should be shown.

Indicates all text should be obscured.

Indicates all text should be obscured, but the last character is visible for a short delay.